components/dashboard/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
Disabling for compilation to improve dx a bit. Once we have our linter enabled for PRs it will catch any unused vars rather than it be a compilation error.